src/tests/ftest/util/ior_test_base.py

```diff
@@ -206,6 +206,10 @@ def run_ior(self, manager, processes, intercept=None):
         try:
             self.pool.display_pool_daos_space()
             out = manager.run()
+
+            for line in out.stdout.splitlines():
+                if 'WARNING' in line:
+                    self.fail("IOR command issued warnings.\n")
```

Comment:
Completely option, but wouldn't a regex be a bit more efficient here?
```
warnings = re.findall(r"WARNING", out.stdout)
if warnings:
    self.fail("IOR command issued warnings:\n  {}".format("\n  ".join(warnings)))
```

Reply:
I think that would print WARNING 49152 times!  I try and avoid using regex unless I have to but yes, that approach would be equally valid.  This code isn't performance critical.